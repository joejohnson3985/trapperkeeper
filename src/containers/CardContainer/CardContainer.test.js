import React from 'react';
import { CardContainer, mapStateToProps} from './index';
import { shallow } from 'enzyme';

describe('CardContainer', () => {
  describe('component', () => {
    let wrapper;

    beforeEach(()=> {
      wrapper = shallow(
        <CardContainer cards={[{},{}]}/>
      )
    })

    it('should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot();    
    });
  })

  describe('mapStateToProps', () => {
    it('should have state from store', () => {
      let mockState = {
        cards: [],
        magic: []
      }
      let expected = {
        cards: []
      }
      const result = mapStateToProps(mockState);
      expect(result).toEqual(expected)
    })
  })
})

