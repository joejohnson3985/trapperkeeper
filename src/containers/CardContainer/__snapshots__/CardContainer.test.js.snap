// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer component should match the snapshot 1`] = `
<div
  className="card-container"
>
  <NavLink
    className="create-card-btn"
    to="/new-note"
  >
    <img
      alt="Create new card plus sign icon"
      className="createNew"
      src="create-new.svg"
    />
  </NavLink>
  <ConnectFunction
    card={Object {}}
  />
  <ConnectFunction
    card={Object {}}
  />
</div>
`;
